src/index.ts

```diff
@@ -87,9 +90,8 @@ export class Marqeta {
     const isForm = isFormData(body)
     // make the appropriate headers
     let headers = {
-      'X_API_KEY': this.apiKey,
       Accept: 'application/json',
-      'X-Peach-Client-Ver': ClientVersion,
```

Comment:
Why not have:

```javascript
      'X-Marqeta-Client-Ver': ClientVersion,
```

this at least allows us to track anything over the wire, if we need to... Right?